julietezekwe/store_manager_reactRedux
# #162980053: React App setup for store manager

### What does this PR do?
It sets up react app with webpack

## Description of task to be completed
- setsup react dom and webpack configuration
- Setup babel compiler
- Sets up server to run on port 8080
- Configures babel for trans piling ECMA Script higher than ES5
- Adds a gitignore file
- Adds a start script to the application

## How should this be manually tested

1. Clone the repo and checkout to devlope then `npm i` to install dependencies
2. Start the app after cloning with `npm run start`
2. The app should run on port 8080
3. Start the server on postman `http://localhost:8080`

## What are the relevant Stories
https://www.pivotaltracker.com/story/show/162980053
